src/Nri/Ui/Carousel/V2.elm

```diff
@@ -0,0 +1,344 @@
+module Nri.Ui.Carousel.V2 exposing
+    ( viewWithPreviousAndNextControls
+    , viewWithTabControls
+    , viewWithCombinedControls
+    , Role(..)
+    )
+
+{-| Patch changes:
+
+  - added new carousel APIs (with tabbed controls/previous and next controls, and combined controls) using [W3C guidelines](https://www.w3.org/WAI/ARIA/apg/patterns/carousel/)
+
+@docs viewWithPreviousAndNextControls
+@docs viewWithTabControls
+@docs viewWithCombinedControls
+@docs Role
+
+-}
+
+import Accessibility.Styled.Aria as Aria
+import Accessibility.Styled.Key
+import Accessibility.Styled.Role as Role
+import Css exposing (..)
+import Html.Styled as Html exposing (..)
+import Html.Styled.Attributes as Attrs exposing (id)
+import List.Extra
+import Nri.Ui.ClickableSvg.V2 as ClickableSvg
+import Nri.Ui.Svg.V1 exposing (Svg)
+import TabsInternal.V2 as TabsInternal
+
+
+{-| `Role`, which can be either [Group](https://w3c.github.io/aria/#group) or [Region](https://w3c.github.io/aria/#region)
```

Comment:
I haven't looked back at the authoring practices guide or experimented at all, and I'm noticing that looking at the docs/examples, I don't know which Role to use.

If we want devs/designers to make a choice here, can we provide some more guidance and add the options to the configurable settings on the example page? Or alternatively, can we pick an approach?

(LMK if you want me to do more research and provide a suggestion here! Out of time for today, but perfectly willing to look into this more if desired.)

Reply:
I added more context about both of them in the docs!